csaf-import/build.gradle.kts

```diff
@@ -1,3 +1,13 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 plugins {
     id("buildlogic.kotlin-library-conventions")
-}
\ No newline at end of file
+}
+
+tasks.withType<KotlinCompile> {
+    dependsOn(rootProject.tasks.getByName("generate"))
+}
+
+sourceSets.main {
+    java.srcDirs("../build/generated-sources/kotlin")
```

Comment:
would move everything generated into `usaf-schema` and then depend on that project from `csaf-import`